org.sonarlint.eclipse.core/src/org/sonarlint/eclipse/core/internal/server/IServer.java

```diff
@@ -115,4 +115,5 @@
 
   boolean isSonarCloud();
 
+  boolean enableNotifications();
```

Comment:
Shouldn't it be name areNotificationEnabled()? The current name make me think this is to enable notifications on this server.